### ISSUE TITLE
not able to scroll windows

### ISSUE THREAD

**Issue by cosimopp** [philc/vimium#4060]
I was watching a YT playlist; I was curious to see other videos by scrolling in the playlist, and yet once I pressed "f" there was no letter for the playlist scrolling bar :(

![vimiumProb png2](https://user-images.githubusercontent.com/95437148/169716749-aa3ae997-c75e-4e1a-855b-92c9860cd5fc.png)
 
It's similar to #4048 : there should be a collection/set of html tags that we can click on and use them.
I noticed the same behavior with scrollable text area as well

Using chrome

**Comment by gdh1995** [comment]
You may use `LinkHints` to select the marker named `SM`,  and it refers the whole list. And then `j` and `k` will scroll the list.

**Comment by cosimopp** [comment]
Yes, it works. But once I "selected" the whole list, I'm not able to go back and scroll YT page in order to see the description and comments. Tried ```gF``` but no way

**Comment by gdh1995** [comment]
Then https://github.com/philc/vimium/pull/4016 will help - with it you can select any element to scroll it or its parents.

**Comment by cosimopp** [comment]
I'm sorry, but what do you mean by adding action=hover-and-scroll to LinkHints

**Comment by gdh1995** [comment]
With the PR and `map Fs LinkHints.activateMode action=hover-and-scroll`, you may select any other hint marker, like `JE` and `JH`, to setup the target element of Vimium commands like `j` and `k`.

**Comment by cosimopp** [comment]
I'm feeling quite dummy. What is PR? In a nutshell, what do I have to add in custom key mappings? I'm really sorry

**Comment by gdh1995** [comment]
Um. "PR" means:
* someone writes some code, and adds a "pull request" (here it's #4016) to add the code into the official project (here it's "philc/vimium")
* the request is waiting for reviews from project maintainers (here there's the user named "Philc")
* maintainers can decide whether to accept the request or not; and if they click "accept", then we say the request is "`merged`"

In short:
* currently Vimium can not do what I said.
* if my #4016 is accepted, and then Philc releases a newer version of Vimium with the PR's code, then
  * `map Fs LinkHints.activateMode action=hover-and-scroll` will begin to take effect
  * And then you may press `Fs`, `j` and `e` to select the `JE` hint marker and press `j` / `k` to scroll the whole page
